doc/doc.md

```diff
@@ -166,8 +167,82 @@ export const settings = {
+| type (\*)    | `"boolean"` |           | Defines a param of type 'boolean'                                                                                 |
+| default (\*) | `string`    |           | Default to this value.                                                                                            |
+| editable     | `boolean`   | true      | If false, disables the field in the UI.                                                                           |
+| validation   | `function`  | undefined | If present, executes function with the new value. Should return a string describing the error or null if no error |
+
+#### Color
+
+| Prop         | Type                                                      | Default   | Description                                                                                                       |
+| ------------ | --------------------------------------------------------- | --------- | ----------------------------------------------------------------------------------------------------------------- |
+| type (\*)    | `"color"`                                                 |           | Defines a param of type 'color'                                                                                   |
+| default (\*) | `string`                                                  | undefined | Default to this value.                                                                                            |
+| model        | `rgba\|hex`                                               | `rgba`    | The color model to be used                                                                                        |
+| editable     | `boolean`                                                 | true      | If false, disables the field in the UI.                                                                           |
+| options      | `['value']` or `[{label,value}]` or `{label: value, ...}` | undefined | If present, displays a dropdown with the provided options.                                                        |
+| validation   | `function`                                                | undefined | If present, executes function with the new value. Should return a string describing the error or null if no error |
+
+#### Image
+
+| Prop       | Type       | Default   | Description                                                                                                                   |
+| ---------- | ---------- | --------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| type (\*)  | `"image"`  |           | Defines a param of type 'image'                                                                                               |
+| default    | `string`   | undefined | If present, defaults to this value.                                                                                           |
```

Comment:
I wouldn't list it here. There is no value you could give it for it to work right now.

Reply:
👍